src/EditorFeatures/Test/EditAndContinue/EditAndContinueWorkspaceServiceTests.cs

```diff
@@ -3967,13 +3967,13 @@ void F()
             EnterBreakState(debuggingSession, GetActiveStatementDebugInfosCSharp(
                 new[] { GetGeneratedCodeFromMarkedSource(markedSource1) },
                 filePaths: new[] { generatedDocument1.FilePath },
```

Comment:
💭 Interesting that these weren't included

Reply:
yeah.  not sure why.  will investigate.

Reply:
works in tests: https://github.com/dotnet/roslyn/pull/71821

my guess is that this was something fixed around nullable+arrays that hasn't been rolled into the latest sdk yet.